Documentation/ProjectFile/prj/process_variables/process_variable/t_initial_condition.md

```diff
@@ -1 +1,3 @@
-\ogs_missing_documentation
+Initial condition for the process variable.
+
+This setting references a \ref ogs_file_param__parameter "parameter" by name.
```

Comment:
> initial solution of the linear solver

At least with LIS you can disable that. (I don't know, if you can really disable it in OGS because of our default LIS option); and for the Newton-Raphson method the statement is not true.

For the nonlinear solver IMHO it is natural that the IC is really used as the initial value. Otherwise you would assemble a (possibly) completely meaningless equation system.